primitives/runtime/src/lib.rs

```diff
@@ -434,6 +434,22 @@ impl traits::Printable for DispatchError {
 	}
 }
 
+/// This type describes the result of a module function call which is called through
```

Comment:
nit: I'd put these guys next to `SimpleDispatcher`.

Reply:
Or not. There's a lot more dispatch-related and sadly the ordering is a bit confused. nvmd.